templates/r10k-configmap.yaml

```diff
@@ -1,67 +0,0 @@
-{{- if .Values.puppetserver.puppeturl }}
```

Comment:
The file should be moved with `git move` - and not deleted. No changes can be compared right now.

Reply:
I thought I had done that, but will see what I can do to correct it.

Reply:
If you cannot - no worries. I'll check it line by line. It'll be very good in the future to make sure we follow that practice - will save both of us time. Thanks!